wiki/Legal/Copyright/Music_Licensing/en.md

```diff
@@ -0,0 +1,23 @@
+---
+legal: true
+---
+
+# Music Licensing
+
+Artists who enter a licensing agreement with osu! are listed under the [Featured Artists](/wiki/Featured_Artists) program. The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game.
+
+Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.
+
+## Terms
+
+Music licensed by osu! is freely available for download from the [Featured Artists listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are level (beatmap) templates. A one minute automatically generated preview of each track is available for streaming on its artist listing. Because songs are used as a catalog for community members to create beatmaps, some of them may not be immediately used as in-game content.
```

Comment:
```suggestion
Music licensed by osu! is freely available for download from the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are level (beatmap) templates. A one-minute automatically generated preview of each track is available for streaming on its artist's page. Because songs are used as a catalog for community members to create beatmaps, some of them may not be immediately used as in-game content.
```